.github/workflows/test-pr.yaml

```
@@ -51,13 +52,29 @@ jobs:
             "ubuntu:jammy",
             "debian:11",
             "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",
```

```suggestion
            "mcr.microsoft.com/devcontainers/base:ubuntu",
```

We do have this image --> https://mcr.microsoft.com/v2/devcontainers/base/tags/list